modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -195,7 +195,8 @@ const setupPurify = (settings: DomParserSettings, schema: Schema): DOMPurifyI =>
 };
 
 const transferChildren = (parent: AstNode, nativeParent: Node) => {
-  const isSpecial = specialElements.has(parent.name);
+  // Exclude textarea and title as their content needs to be parsed as RCDATA
+  const isSpecial = specialElements.has(parent.name) && parent.name !== 'textarea' && parent.name !== 'title';
```

Comment:
If this is a special subset of elements that need special treatment should they be their own set instead like. `specialRcDataElements` or something. Are textarea and title the only ones. Could we put a spec link in the comment?

Reply:
Have created a variable for it as requested. The spec link was already next to the variable